<launch>
    <!-- 
      In this file, "pepper" is referred to as the robot's name
      in DNS or the /etc/hosts file and points to the IP of the robot.
    --> 

	<arg name="robot_ip" default="pepper" />
  	<arg name="robot_port" default="$(optenv NAO_PORT 9559)" />
  	<arg name="robot_name" default="pepper" />
  	
 	<param name="robot_ip" type="string" value="$(arg robot_ip)" />
 	<param name="robot_port" type="string" value="$(arg robot_port)" />
 	<param name="robot_name" type="string" value="$(arg robot_name)" />  

	<arg name="engMode" default="FullyEngaged" />  
	<!--arg name="engMode" default="SemiEngaged" /-->  
	<arg name="trackMode" default="BodyRotation" />
	<!--arg name="trackMode" default="MoveContextually" /-->
	<arg name="tabShow" default="True" />
 	
 	<param name="engMode" type="string" value="$(arg engMode)" />  
	<param name="trackMode" type="string" value="$(arg trackMode)" />  
	<param name="tabShow" type="string" value="$(arg tabShow)" />  

  	<!-- Launch DIALOG -->
  	<node pkg="w_icar" type="w_icar.py" name="$(arg robot_name)_w_icar" output="screen"/>
  	
</launch>